### ISSUE TITLE
FOREGROUND_SERVICE_DATA_SYNC

### ISSUE THREAD

**Issue by vishalyad16** [Rapsssito/react-native-background-actions#248]
Not able to release app playstore blocking the release requesting for a video with foreground service, please help with solution! BTW why do we need FOREGROUND_SERVICE_DATA_SYNC??

**Comment by SurajMohanty02** [comment]
I'm also unable to release the apk to playstore due to FOREGROUND_SERVICE_DATA_SYNC
FOREGROUND_SERVICE_LOCATION . Please help me.

**Comment by SurajMohanty02** [comment]
@vishalyad16 Did you find any solution?

**Comment by patphantuan251** [comment]
When releasing an app with FOREGROUND_SERVICE_DATA_SYNC permission to the store, PlayStore will reject it and ask for a screen recording video and a description of why you use FOREGROUND_SERVICE_DATA_SYNC permission. Follow this and the app will be approved.

**Comment by vishalyad16** [comment]
@SurajMohanty02 

Yes, I have found a solution. I have created an individual video explaining why we are using it.

**Comment by enact-on** [comment]
@vishalyad16 

Could you please let me know what exactly have you explained in the video. Mine too got rejected two days ago citing same reason.

Thanks

**Comment by Rapsssito** [comment]
Thanks to #265, you are now able to set `<uses-permission>` as desired!
